geomesa-accumulo/geomesa-accumulo-distributed-runtime/pom.xml

```diff
@@ -28,24 +28,6 @@
             <artifactId>geomesa-accumulo-datastore_2.11</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-accumulo-raster_2.11</artifactId>
```

Comment:
I think it would work with the raster jar there but I've changed this branch to build a shaded runtime raster jar in addition to the runtime jar. I think it makes more sense from a user experience continuity perspective and it makes the install scripts easier.